libbpf-tools/sigsnoop.bpf.c

```diff
@@ -142,6 +151,13 @@ int sig_trace(struct trace_event_raw_signal_generate *ctx)
 	if (filtered_pid && pid != filtered_pid)
 		return 0;
 
+	ttask = bpf_task_from_pid(tpid);
```

Comment:
Let's do this only when this kfunc exists.

Reply:
Fix it , thanks, please review.